src/stories/components/StatusChipLegacy/StatusChipLegacy.tsx

```diff
@@ -54,14 +54,16 @@ const colors: { [id: string]: any } = {
     backgroundDark: '#FF8E3633',
   },
 };
-
-export const StatusChip = (props: StatusChipProps) => {
+/**
+ * @deprecated use StatusChip  instead
```

Comment:
Are those two statuses the same thing? the legacy status can be 4 variants only and this includes "Obsolete" which is not a budget statement status. Maybe we should  handle the new one as other thing (including a different name if they're different) (Note that the new one looks like a CU/EA status

Reply:
you are right, done